### PR TITLE
Proposal to decrease the log level for some messages

### PR DESCRIPTION
**Pull request by pguillard**
Proposition en rapport à ce ticket chez RueDuCommerce : https://jira.rueducommerce.fr/browse/DRU-605

### CODE REVIEW

File: src/Queue/Queue.php [hunk 1]
```diff
@@ -132,7 +132,7 @@ public function claimItem($lease_time = 3600) {
    *   An item returned by DrupalQueueInterface::claimItem().
    */
   public function deleteItem($item) {
-    $this->logger->info('Item @id acknowledged from @queue', [
+    $this->logger->debug('Item @id acknowledged from @queue', [
```
Comment (fgm) [inline src/Queue/Queue.php]: Actually, it's more than acknowledged: it's deleted.

**Review by fgm — APPROVED** [review]
Good for me. Could take advantage of this PR to fix the word choice on deleteItem.

**Comment by pguillard** [comment]
Done

**Review by fgm — APPROVED** [review]
Good for me, but this is not my repo: ask @phcorp to merge it then submit on my repo, which is the one we're using. Or submit the same PR on my repo.

**Comment by phcorp** [comment]
Merged. Thank you!

**Comment by pguillard** [comment]
Thank you too !

**Pull request merged**